### PR TITLE
Fix PHP CAS client download URL

### PR DESCRIPTION
**Pull request by xdmnl**

### CODE REVIEW

**Comment by jtgasper3** [comment]
Thanks for the PR. I've got a question...

File: php-cas/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN yum -y update \
     && yum -y install httpd mod_ssl php php-pear php-xml php-pdo wget \
     && yum -y clean all
 
-RUN pear install http://downloads.jasig.org/cas-clients/php/current.tgz
+RUN pear install https://developer.jasig.org/cas-clients/php/current.tgz
```
Comment (jtgasper3) [inline php-cas/Dockerfile:9]: Is there a formal difference between the two hostnames? Both url's work for me.

Comment (jtgasper3) [inline php-cas/Dockerfile:9]: Ahhh, I just noticed the new one runs under https... That's a bonus.

**Comment by jtgasper3** [comment]
Thanks for the better url.

File: php-cas/Dockerfile (hunk 1 above)
Comment (xdmnl) [inline php-cas/Dockerfile:9]: Yesterday, the URL didn't work for me so it was impossible to `docker-compose build`.
Thanks for the quick merge!

**Pull request merged**